packages/edit-post/src/components/header/header-toolbar/index.js

```diff
@@ -68,11 +75,48 @@ function HeaderToolbar() {
 		: /* translators: accessibility text for the editor toolbar when Top Toolbar is off */
 		  __( 'Document tools' );
 
+	const overflowItems = (
+		<>
+			{ isLargeViewport && (
+				<ToolbarItem
+					as={ ToolSelector }
+					showTooltip={ ! showIconLabels }
+					isTertiary={ showIconLabels }
+				/>
+			) }
+			<ToolbarItem
+				as={ EditorHistoryUndo }
+				showTooltip={ ! showIconLabels }
+				isTertiary={ showIconLabels }
+			/>
+			<ToolbarItem
+				as={ EditorHistoryRedo }
+				showTooltip={ ! showIconLabels }
+				isTertiary={ showIconLabels }
+			/>
+			<ToolbarItem
+				as={ TableOfContents }
+				hasOutlineItemsDisabled={ isTextModeEnabled }
+				repositionDropdown={ showIconLabels && ! isWideViewport }
+				showTooltip={ ! showIconLabels }
+				isTertiary={ showIconLabels }
+			/>
+			<ToolbarItem
+				as={ BlockNavigationDropdown }
+				isDisabled={ isTextModeEnabled }
```

Comment:
The disabled items in the overflow menu don't seem to be consistent with other dropdown menus:
<img width="281" alt="Screenshot 2020-08-13 at 4 01 11 pm" src="https://user-images.githubusercontent.com/677833/90109827-ed97fc80-dd7e-11ea-9043-cabe6d7e48fc.png">

Reply:
Yeah, they are meant to be consistent with their out-of-overflow state 😅 but perhaps @mapk can comment further on that.